src/openforms/appointments/tests/test_endpoints.py

```diff
@@ -0,0 +1,123 @@
+import uuid
+from datetime import datetime, timezone
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.urls import reverse
+
+from furl import furl
+
+from openforms.config.models import GlobalConfiguration
+from openforms.submissions.constants import SUBMISSIONS_SESSION_KEY
+from openforms.submissions.tests.factories import SubmissionFactory
+
+from .factories import AppointmentInfoFactory
+
+
+class VerifyCancelAppointmentLinkViewTests(TestCase):
+    @patch(
+        "openforms.appointments.views.submission_appointment_token_generator.check_token",
+        return_value=True,
+    )
+    def test_good_token_and_submission_redirect_and_add_submission_to_session(
+        self, check_token_mock
+    ):
+        config = GlobalConfiguration.get_solo()
+        config.cancel_appointment_page = "http://maykinmedia.nl/afspraak-annuleren"
+        config.save()
+
+        submission = SubmissionFactory.create()
+        AppointmentInfoFactory.create(
+            submission=submission,
+            start_time=datetime(2021, 7, 21, 12, 00, 00, tzinfo=timezone.utc),
+        )
+
+        endpoint = reverse(
+            "appointments:appointments-verify-cancel-appointment-link",
+            kwargs={
+                "token": "mocked",
+                "submission_uuid": submission.uuid,
+            },
+        )
+
+        response = self.client.get(endpoint)
+
+        # Assert submission is stored in session
+        self.assertIn(
+            str(submission.uuid), self.client.session[SUBMISSIONS_SESSION_KEY]
+        )
+
+        expected_redirect_url = (
+            furl(config.cancel_appointment_page)
```

Comment:
use string value from earlier set up to make it more explicit - in general you want to avoid "best practices" in tests and keep them as dumb and straight forward as possible.